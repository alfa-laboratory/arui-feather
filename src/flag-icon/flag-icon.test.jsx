/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from 'react';
import { mount } from 'enzyme';

import FlagIcon from './flag-icon';

describe('flag-icon', () => {
    it('renders without problems', () => {
        let flagIcon = mount(<FlagIcon />);
        expect(flagIcon).toMatchSnapshot();
    });
});
