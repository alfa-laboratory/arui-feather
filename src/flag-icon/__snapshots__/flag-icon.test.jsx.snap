// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`flag-icon renders without problems 1`] = `
<FlagIcon
  isFlat={false}
  mode="chunk"
  size="m"
>
  <span
    className="flag-icon flag-icon_mode_chunk flag-icon_size_m flag-icon_theme_alfa-on-white"
  />
</FlagIcon>
`;
